src/include/catalog/database_catalog.h

```diff
@@ -253,8 +257,8 @@ class DatabaseCatalog {
    * @param class_oid oid of table or index
    * @return the column from pg_attribute
    */
-  template <typename Column, typename ClassOid, typename ColOid>
-  std::unique_ptr<Column> GetColumn(transaction::TransactionContext *txn, ClassOid class_oid, ColOid col_oid);
+  // template <typename Column, typename ClassOid, typename ColOid>
+  // Column GetColumn(transaction::TransactionContext *txn, ClassOid class_oid, ColOid col_oid);
```

Comment:
Why is this commented out?

Reply:
As best as I could tell, there is no actual use-case for materializing a single column so I commented it out during the update.  I've now completely removed it.